src/kemal/handler.cr

```diff
@@ -15,6 +15,7 @@ class Kemal::Handler < HTTP::Handler
 
   def add_route(method, path, &handler : Kemal::Context -> _)
     @routes << Route.new(method, path, &handler)
+    @routes << Route.new("HEAD", path, &handler) if method == "GET"
```

Comment:
@sdogruyol: You are absolutely right and this is most obvious way to add `HEAD` for defined `GET` routes. Servers are implemented in such a way that there is no need to manually clear `response.body`.